host-bmc/host_pdr_handler.cpp

```diff
@@ -726,7 +724,7 @@ void HostPDRHandler::parseStateSensorPDRs()
         }
         // If there is no mapping for terminusHandle assign the reserved TID
         // value of 0xFF to indicate that.
-        catch (const std::out_of_range& e)
+        catch (const std::out_of_range& /*e*/)
```

Comment:
Please remove /*e*/. Its applicable to all the below files where you have done in similar way.

Reply:
updated as per comment.